templates/endpoints/README.md

```diff
@@ -151,6 +153,60 @@ Endpoints allows you to easily add new models by adding a single configuration f
 To learn more about how to customize or add new models, 
 see the [Model Registry](models/README.md).
 
+# Serving LoRA Models
+
+`serve_lora.yaml` and `query_lora.py` are provided for you in this template.
+- If you just want to deploy static LoRA models, you can list them in `multiplex_lora_adapters` config in the `serve_lora.yaml` file.
+- If you want to dynamically loading LoRA models to a running service, you can use `dynamic_lora_loading_path` config in the `serve_lora.yaml` file.
+- Both can be set at the same time and complement to each other.
```

Comment:
```suggestion
- Both can be set at the same time and complement to each other. The lora adapters specified in "multiplex_lora_adapters" will be always loaded in each serve replica whereas the ones from "dynamic_lora_loading_path" will be loaded as needed.
```
Is this accurate?

Reply:
Actually I think both are only loaded when the user made the call to the model 😅

Reply:
ok , then we should clarify that the adapters are only loaded when the model is called

Reply:
This makes the "dynamic" vs "static" wording confusing.